docs/integrations/sources/mixpanel.md

```diff
@@ -51,7 +51,8 @@ Syncing huge date windows may take longer due to Mixpanel's low API rate-limits
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                     |
 |:--------|:-----------|:---------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
-| 0.1.39  | 2023-09-15 | [30469](https://github.com/airbytehq/airbyte/pull/30469) | Add default primary key `distinct_id` to `Export` stream   |
+| 1.0.0   | 2023-09-20 | [30149](https://github.com/airbytehq/airbyte/pull/30149) | Prepare for certifying to the next stage                                                                    |
```

Comment:
Can we describe what changed in the actual code instead of referring to certification?